### ISSUE TITLE
Changes to default / help text

### ISSUE THREAD

**Issue by niryariv** [hossted/cli#37]
under "Available Commands" use the following:

```
Available Commands:
  register    [r] Register your application with the hossted ecosystem
  set auth false|true     Enable / disable HTTP Basic Auth
  set remote-support false|true   Enable / disable ssh access for Hossted support team
  set domain <domain>  Set a custom domain 
  help        Help about any command
  logs        [l] View Application logs
  ps          [ps] docker-compose ps of the application
  version     [v] Get the version of the hossted CLI program
```

The main motivations are: show the full set commands, so users know what to do with them, unlist completion until we have it done right, move register / set to the top as they are the most useful

**Comment by billylkc** [comment]
Currently the available comments are generated by the framework. Need to study whether the framework allows to change the text there. Not really recommend to change it tho.

**Comment by niryariv** [comment]
Research our options

**Comment by billylkc** [comment]
Should be available in 0.2.2. Please check.

Test: Able to see custom default help text with **hossted**, should be able to see important subcommands (e.g. hossted set domain) in the new help text.
